model/jpa/src/main/java/org/keycloak/models/jpa/JpaUserProvider.java

```diff
@@ -692,6 +692,7 @@ public int getUsersCount(RealmModel realm, Map<String, String> params, Set<Strin
 
         List<Predicate> restrictions = predicates(params, root);
         restrictions.add(cb.equal(root.get("realmId"), realm.getId()));
+        // restrictions.add(from.get("groupId").in(groupIds)); ???
```

Comment:
Please remove. If there is some issue with the current query, please open a separate PR.